apps/prairielearn/assets/scripts/lib/questionsTable.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-env browser, jquery */
+
+import _ from 'lodash';
+import { onDocumentReady, decodeData } from '@prairielearn/browser-utils';
+import { html } from '@prairielearn/html';
+
+onDocumentReady(() => {
+  const { course_instance_ids, showAddQuestionButton, urlPrefix, plainUrlPrefix } =
+    decodeData('questions-table-data');
+  window.topicList = function () {
+    var data = $('#questionsTable').bootstrapTable('getData');
+    return _.keyBy(_.map(data, (row) => row.topic.name));
+  };
+
+  window.tagsList = function () {
+    var data = $('#questionsTable').bootstrapTable('getData');
+    return _.keyBy(_.map(_.flatten(_.filter(_.map(data, (row) => row.tags))), (row) => row.name));
+  };
+
+  window.versionList = function () {
+    var data = $('#questionsTable').bootstrapTable('getData');
+    return _.keyBy(_.map(data, (row) => row.display_type));
+  };
+
+  window.qidFormatter = function (qid, question) {
+    var text = '';
+    if (question.sync_errors) {
+      text += html`<button
+        class="btn btn-xs mr-1"
+        data-toggle="popover"
+        data-title="Sync Errors"
+        data-content='<pre style="background-color: black" class="text-white rounded p-3">${_.escape(
+          question.sync_errors_ansified,
+        )}</pre>'
```

Comment:
This looks to be untested? You're double-escaping the data here:

<img width="2050" alt="Screenshot 2023-09-28 at 15 13 16" src="https://github.com/PrairieLearn/PrairieLearn/assets/1476544/cf4c9fc7-1029-48c1-8233-6d354e64ec2a">

Reply:
The issue is not double-escaping, is escaping when it doesn't need to be. The ansified version includes HTML formatting, so it should not be escaped.

Reply:
fixed, thanks